alacritty_terminal/src/selection.rs

```diff
@@ -579,7 +579,7 @@ mod test {
         assert_eq!(selection.to_span(&term(5, 10)).unwrap(), Span {
             start: Point::new(0, Column(4)),
             end: Point::new(2, Column(4)),
-            is_block: true,
+            is_block: true
```

Comment:
Rustfmt needs to be run with nightly to include the nightly formatting options. That's why it has removed the trailing semicolons here for you. Please undo these changes either manually or by running rustfmt with a nightly compiler.

Reply:
Hmm, I was running with nightly:

```
zsh/4 9472  (git)-[live-reload-font-size]-% rustup update nightly-x86_64-unknown-linux-gnu
info: syncing channel updates for 'nightly-x86_64-unknown-linux-gnu'

  nightly-x86_64-unknown-linux-gnu unchanged - rustc 1.40.0-nightly (702b45e40 2019-10-01)

info: checking for self-updates
zsh/4 9473  (git)-[live-reload-font-size]-% rustup which cargo
/home/wayne/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/bin/cargo
zsh/4 9474  (git)-[live-reload-font-size]-% /home/wayne/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/bin/cargo fmt
zsh/4 9475  (git)-[live-reload-font-size]-% git diff
zsh/4 9476  (git)-[live-reload-font-size]-%
```

> Please undo these changes either manually or by running rustfmt with a nightly compiler.

I'd be happy to remove them but I want to make sure we have a common understanding that I was using nightly. Also, it seems like the next person to run `cargo fmt` will produce the same changes anyway.

Reply:
Do you think it would be useful to add a travis check that fails if `rustfmt --check` returns non-zero?

Reply:
We have this check on CI, it's optional though, because we're running it on nightly I guess.

Reply:
Ah I didn't think to look at what was already running in travis. Looks like the `rustfmt` check is currently passing: https://travis-ci.org/jwilm/alacritty/jobs/592595414

Reply:
Make sure to install rustfmt through `rustup` to make sure it's pulling the correct version.

Reply:
That is how I have it installed. One possibility here is that I had an older version of `nightly` installed before this discussion and I updated it specifically to try to fix this; it may be that the older `nightly` was more opinionated about commas whereas the new nightly will allow either style. Whatever the case, I manually reverted the changes in a newer commit.